src/new-client/src/App/Trades/TradesGrid/TradesGrid.tsx

```diff
@@ -8,6 +8,7 @@ import {
 import { capitalize } from "utils/capitalize"
 import { format as formatDate } from "date-fns"
 import { useTableTrades, sortFieldSelections$, useTableSort } from "../services"
+import { FixedSizeList as List } from "react-window"
```

Comment:
In our case, I would import as "VirtualizedTable" so that it reads semantically for what it is doing